src/Microsoft.Data.SqlClient/tests/tools/Microsoft.Data.SqlClient.TestUtilities/Fixtures/CertificateFixtureBase.cs

```diff
@@ -166,15 +184,23 @@ exit 1
                 }
 
                 // Process completed successfully if it had an exit code of zero, the command output will be the base64-encoded certificate
-                if (psProcess.ExitCode == 0)
+                var code = psProcess.ExitCode;
+                if (code == 0)
                 {
                     return new X509Certificate2(Convert.FromBase64String(commandOutput), password);
                 }
-                else
-                {
-                    throw new Exception($"PowerShell command raised exception: {commandOutput}");
-                }
+                
+                Console.WriteLine(
+                    $"PowerShell command failed with exit code {code} on " +
+                    $"attempt {attempt} of {retries}; " +
+                    $"retrying in {delay} seconds...");
+                
+                System.Threading.Thread.Sleep(delay * 1000);
```

Comment:
Two nits:
* If you add a using up top, you don't need the full System.Threading...
* Might be clearer to use TimeSpan.FromSeconds(delay)